### DISCUSSION TITLE
Installation of moveHMM fails on Posit RStudio Pro - Ubuntu

### DISCUSSION THREAD

**Discussion by pabcisa** [TheoMichelot/moveHMM#42] [General]
Hi, I’m encountering an issue when trying to install moveHMM on a Posit RStudio Pro setup running on Ubuntu. All the dependencies were installed correctly, and the package installs without issue on my Windows PC. However, on RStudio Pro, I keep getting the following error (attached full error) during the installation process:

`install.packages("moveHMM", repos = "https://cran.rstudio.com", dependencies = TRUE)`
...
/usr/bin/ld: cannot find Setting: No such file or directory
/usr/bin/ld: cannot find rstudiopm01: No such file or directory
/usr/bin/ld: cannot find repository: No such file or directory
collect2: error: ld returned 1 exit status
make: *** [/opt/R/4.4.0/lib/R/share/make/shlib.mk:10: moveHMM.so] Error 1

Any guidance on what might be causing this issue or how to resolve it would be greatly appreciated.
Thank you!
[installation_error_moveHMM.txt](https://github.com/user-attachments/files/18075592/installation_error_moveHMM.txt)

**Comment by TheoMichelot** [comment]
I'm not sure why you're getting this error; I've never had issues installing the package on Ubuntu before. 

A couple of things you could check:

1. Are you able to install other packages based on Rcpp/RcppArmadillo? ([See "Reverse linking to" list at this link](https://cran.r-project.org/web/packages/RcppArmadillo/index.html).) 
2. Are you able to install moveHMM on that machine in the terminal directly (rather than through RStudio)? You can open the terminal application, run "R", and then paste the install.packages() command you were using.
